sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/KafkaCosmosConfig.java

```diff
@@ -68,6 +68,15 @@ public class KafkaCosmosConfig extends AbstractConfig {
     private static final String AAD_CLIENT_SECRET_DISPLAY = "The client secret/password of the service principal.";
     private static final String DEFAULT_AAD_CLIENT_SECRET = Strings.Emtpy;
 
+    private static final String AAD_AUTH_ENDPOINT_OVERRIDE = "azure.cosmos.auth.aad.authEndpointOverride";
+    private static final String AAD_AUTH_ENDPOINT_OVERRIDE_DOC = "Overrides the Azure Active Directory (AAD) authentication endpoint. "
+        + "This is useful when the Cosmos DB account resides in a non-public Azure cloud environment such as Azure China, Azure US Government, or Azure Germany. "
+        + "By default, the SDK uses the standard AAD endpoint for the public Azure cloud. Set this value if your deployment requires a custom authority URI "
+        + "(e.g., https://login.chinacloudapi.cn/).";
```

Comment:
Just curious if we need to scrub this off - `AzureChina` already maps to above endpoint.